packages/EasyCore/src/Helpers/TrimStringsRecursive.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyCore\Helpers;
+
+final class TrimStringsRecursive implements CleanerInterface
```

Comment:
Let's name it as RecursiveStringsTrimmer

Reply:
Done.